Civi/Funding/PayoutProcess/Validator/DrawdownValidator.php

```diff
@@ -63,6 +67,27 @@ public function __construct(FundingCaseManager $fundingCaseManager, PayoutProces
    * phpcs:disable Drupal.Commenting.FunctionComment.IncorrectTypeHint
    */
   public function validate(AbstractEntity $new, AbstractEntity $current): EntityValidationResult {
+    $payoutProcess = $this->getPayoutProcess($new);
+    $this->assertNotClosed($payoutProcess);
+
+    if ($new->getAmount() < 0) {
+      $payoutProcess = $this->getPayoutProcess($new);
```

Comment:
no necessary, `$payoutProcess` is already set (line 70)

Reply:
Changed.